src/test/java/com/example/codebase/controller/MemberControllerTest.java

```diff
@@ -467,15 +465,51 @@ void create_curator() throws Exception {
         dto.setSnsUrl("https://localhost/");
         dto.setCompanyName("회사 이름");
         dto.setCompanyRole("기획자");
-        dto.setUsername(member.getUsername());
-
-        mockMvc.perform(
-                        post("/api/members/curator")
-                                .contentType(MediaType.APPLICATION_JSON)
-                                .content(objectMapper.writeValueAsString(dto))
-                )
-                .andDo(print())
-                .andExpect(status().isCreated());
-    }
-
-}
\ No newline at end of file
+    dto.setUsername(member.getUsername());
+
+    mockMvc
+        .perform(
+            post("/api/members/curator")
+                .contentType(MediaType.APPLICATION_JSON)
+                .content(objectMapper.writeValueAsString(dto)))
+        .andDo(print())
+        .andExpect(status().isCreated());
+  }
+
+  @DisplayName("이메일로 유저 리스트 조회")
+  @Test
+  void 이메일로_유저_리스트_조회() throws Exception {
+    Member member = createOrLoadMember(2);
+
+    mockMvc
+        .perform(get("/api/members/search/{email}", member.getEmail()))
+        .andDo(print())
+        .andExpect(status().isOk());
+  }
+
+  @DisplayName("이름으로 유저 리스트 조회")
```

Comment:
유저이름 조회 테스트인데 테스트명이 잘못된 것 같습니다.

```suggestion
  @DisplayName("유저이름으로 유저 리스트 조회")
```
로 변경하는게 어떨까요